### PR TITLE
Assignment5 @DakotaCondos

### PR DESCRIPTION
**Pull request by mpennock5**
Website: https://polite-dune-09ecd591e.1.azurestaticapps.net/

Api Endpoints :https://wapiclass.azurewebsites.net/swagger/index.html

### CODE REVIEW

**Comment by dcox2005** [comment]
api gives this error: SqlException: Column names in each table must be unique. Column name 'Common' in table 'Words' is specified more than once.
You may need to delete your database on azure and rebuild it with the same name.  If this doesn't work, maybe also double check your Words.cs data file and see if there was an issue there.

**Comment by PlantBrush** [comment]
As @dcox2005 pointed out, there seems to be an issue with the database, so I can't test the functions of the word editor. The design for the word editor page looks good, but I would suggest using a v-checkbox to toggle whether each word is common. That's a much easier system for clarity and ease of use. While not in the assignment specifications, you could also improve the page by making it more responsive.

**Comment by kelseymcmahon** [comment]
I wasn't able to try out the front end app because of a sql server error in Azure, but the API methods look good. The admin page for the wordle app still needs to make those axios calls to the api for the token so a user can delete and add words. Also your going to need a login page or popup to get those credentials from the user.

**Comment by mpennock5** [comment]
Admin@intellitect.com
YoungAdmin@intellitect.com
P@ssw0rd123 for both

**Comment by mpennock5** [comment]
For some reason the token is getting set in the the JWT.setToken method but the axios header is acting very funky. We cannot get it to send the correct content type, even when manually setting it, and we cannot get the token to consistently stay in the header. It seems that we are doing it exactly as we went over in class but something about axios is not happy. Everything works on our swagger endpoints when it comes to the authorizations but the axios header is behaving unexpectedly. Any feedback on this would be appreciated.

**Review by mpennock5 — COMMENTED** [review]
Had a lot of unexpected issues with the axios calls surrounding the token. Took away from cleaning up all the features on the front end. I left some  additional comments on exactly what we were hung up on and what our workarounds were but if you notice why it was behaving unexpectedly please let us know because we were stumped!

File: wordle-web/pages/admintools.vue [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+                :append-icon="'mdi-plus-circle-outline'"
+                @click:append="addWord(wordToAdd)"
+              ></v-text-field>
+            </div>
+          </v-col>
+        </v-row>
+
+        <v-divider></v-divider>
+
+        <v-pagination
```
Comment (mmwoodfo) [inline wordle-web/pages/admintools.vue:65]: Clicking the arrows on this seems to break the functionality.

File: wordle-web/pages/admintools.vue [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+              <th style="text-align: center">Common</th>
+              <th style="text-align: right">Actions</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr v-for="(word, wordId) in words" :key="wordId">
+              <td>{{ word.item1 }}</td>
+              <td style="text-align: center">{{ word.item2 }}</td>
+              <td style="text-align: right">
+                <v-btn @click="commonFlag(word.item1, true)">common+</v-btn>
```
Comment (mmwoodfo) [inline wordle-web/pages/admintools.vue:88]: As a non-logged in user, I seem to be able to edit this.

**Pull request closed without merging**